tests/test_array.py

```
@@ -39,12 +39,17 @@ async def test_removed_actions_are_invalid(self, _mock_graphql: AsyncMock) -> No
             with pytest.raises(ToolError, match="Invalid action"):
                 await tool_fn(action=action)
 
+    async def test_parity_start_requires_correct(self, _mock_graphql: AsyncMock) -> None:
```

Ruff (0.15.5)

[warning] Fixture `_mock_graphql` without value is injected as parameter, use `@pytest.mark.usefixtures` instead

(PT019)